src/components/Chain.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { ChainColors, ChainName } from '../constants/Chains';
+
+const Wrapper = styled.div<{ $background?: string }>`
+  padding: 0.4rem 0.8rem;
+  border-radius: 20px;
+  background: ${(props) => props.$background ?? `var(--manatee)`};
+  color: var(--white);
+`;
+
+interface Props {
+  className?: string;
+  chain: ChainName;
+  handleClick?: () => void;
```

Comment:
nit: use `React.MouseEventHandler`

```suggestion
  handleClick?: React.MouseEventHandler;
```